ESP32CAM_Car/ESP32CAM_Car.ino

```diff
@@ -19,8 +19,8 @@
 //#define CAMERA_MODEL_M5STACK_PSRAM
 #define CAMERA_MODEL_AI_THINKER
 
-const char* ssid = "AndroidAP";   //Enter SSID WIFI Name
-const char* password = "pftb1416";   //Enter WIFI Password
+const char* ssid = "Roni";   //Enter SSID WIFI Name
```

Comment:
Most of the time we will be using my computer so don't change it to Roni's computer